resources/windows/wix/registryEntries.wxs.erb

```diff
@@ -0,0 +1,99 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <!--
+    Puppet Agent Specific Registry Entries
+    This overrides the generic file in the vanagon directory
+  -->
+  <Fragment>
+    <ComponentGroup Id="RegistryComponentGroup">
+      <Component Id="RegistryEntries" Guid="68F5D91C-49C8-43F7-940A-481007689E79" Win64="no">
+        <RegistryKey
+          Root="HKLM"
+          Key="SOFTWARE\<%= settings[:company_name] %>\<%= settings[:product_name] %>"
+          Action="create" >
```

Comment:
@ferventcoder @Iristyle Question here - the component/registry entry on line 62 has ```Permanent="yes"``` set.
Should we be doing same here ?